src/ViewModel/Converter/ArticleContentHeaderConverter.php

```diff
@@ -30,11 +32,27 @@ public function __construct(ViewModelConverter $viewModelConverter, UrlGenerator
      */
     public function convert($object, string $viewModel = null, array $context = []) : ViewModel
     {
+        $isMagazine = $context['isMagazine'] ?? false;
+
+        $breadcrumb = ($isMagazine || 'feature' === $object->getType()) ? [
```

Comment:
Just curious: why is feature not a magazine type (line 916)?

Reply:
Feature articles fo fall under magazine but the template we wish them to be presented as research articles in all aspects except the breadcrumb.

Reply:
Got it, thanks.

Reply:
I have added a test in [ArticleControllerTest](https://github.com/elifesciences/journal/pull/1612/files#diff-4038dc5ddf83ceda61dd1f6420ea8e5ae31dc7f7b62314e517a551970e57b72d) to verify this behaviour.